docs/framework/network-programming/certificate-selection-and-validation.md

```diff
@@ -35,7 +35,7 @@ The <xref:System.Net> classes support several ways to select and validate <xref:
 ## Tools for Certificate Configuration  
  A number of tools are available for client and server certificate configuration.  
   
- The *Winhttpcertcfg.exe* tool can be used to configure client certificates. The *Winhttpcertcfg.exe* tool is provided as one of the tools with the Windows Server 2003 Resource Kit. This tool is also available as a download as part of the Windows Server 2003 Resource Kit Tools at www.microsoft.com.  
+ The *Winhttpcertcfg.exe* tool can be used to configure client certificates. The *Winhttpcertcfg.exe* tool is provided as one of the tools with the Windows Server 2003 Resource Kit. This tool is also available as a download as part of the Windows Server 2003 Resource Kit Tools at [www.microsoft.com]().  
```

Comment:
The Microsoft URL is missing inside the parens.

Reply:
Whoops I mean to do `<www.microsoft.com>`

Reply:
Actually no I was correct 😄  the commonmark spec says it's OK: https://spec.commonmark.org/0.28/#example-461

Reply:
It looks like the resulting anchor tag's `href` attribute will be empty if you stick with what you have now.

Reply:
ugh you're right. I'm stupid :) I'll fix it